workbench-core/swb-common-ui/src/api/apiHelper.ts

```diff
@@ -7,6 +7,23 @@ import axios, { AxiosRequestConfig } from 'axios';
 
 const urlBase: string | undefined = process.env.NEXT_PUBLIC_API_BASE_URL;
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
not blocking, but might be worth it in the future to look into using generics to get rid of `any` usage

Reply:
or possibly using `unknown` instead of `any`

Reply:
adding a TODO